README.md

```diff
@@ -19,61 +19,61 @@ The PR triggers a series of GitHub actions that will validate the new submitted
 
 Microsoft will respond to a PR with all checks in 3 business days.
 
-## Tools
```

Comment:
The above "Submit a model" should be reviewed/revised since it references areas which have been changed.